modules/labs-react/header/spec/GlobalHeader.spec.tsx

```diff
@@ -1,111 +1,26 @@
 import * as React from 'react';
 import DeprecatedGlobalHeader from '../lib/GlobalHeader';
-import {shallow} from 'enzyme';
-import DeprecatedHeader from '../lib/Header';
-import {SearchForm} from '@workday/canvas-kit-labs-react/search-form';
-import {DeprecatedDubLogoTitle} from '../lib/parts';
-import {DeprecatedHeaderTheme, DeprecatedHeaderVariant} from '../lib/shared/types';
 
-declare global {
-  interface Window {
-    resizeBy: (x: number, y: number) => void;
-  }
-}
-const map: {[key: string]: any} = {};
-window.addEventListener = jest.fn((event, callback) => {
-  map[event] = callback;
-});
-window.removeEventListener = jest.fn((event, callback) => {
-  if (map[event] && map[event] === callback) {
-    delete map[event];
-  }
-});
-window.dispatchEvent = (event: Event) => {
-  if (map[event.type]) {
-    map[event.type]();
-  }
-
-  // TODO: not totally accurate
-  return false;
-};
-
-window.resizeBy = (x: number, y: number) => {
-  // @ts-ignore
-  window.innerWidth = x;
-  // @ts-ignore
-  window.innerHeight = y;
-  window.dispatchEvent(new Event('resize'));
-};
-
-// @ts-ignore
-window.requestAnimationFrame = cbFn => cbFn();
```

Comment:
Luckily none of this is necessary anymore.

Reply:
haha BYE! ✌🏻